audio/openal-soft/Portfile

```diff
@@ -3,12 +3,13 @@
 PortSystem              1.0
 PortGroup               cmake 1.1
 PortGroup               compiler_blacklist_versions 1.0
-if {[variant_isset gui]} {
+if {${subport} eq "openal-soft" && [variant_isset gui]} {
 PortGroup               qt5 1.0
 }
 
 name                    openal-soft
 version                 1.20.0
+revision                2
```

Comment:
I'm not sure why you're increasing the revision to 2 instead of 1 but in any case this revision should only apply when `${subport} eq ${name}`. Each other subport should have its own revision line.

Reply:
> increasing the revision to 2 instead of 1

Oh I see, each commit increased the revision. Fine.

Reply:
Thank you for noticing.
The port and support now have their own revision numbers.